tests/__init__.py

```diff
@@ -1,6 +1,10 @@
 from __future__ import absolute_import, division, print_function
 
-from attr import Attribute
+import string
+
+from hypothesis import strategies as st
+
+from attr import Attribute, ib
```

Comment:
I find using `ib` on itself weird and confusing.

Just import additionally `attr` and use `attr.ib`.